Dockerfile

```diff
@@ -1,24 +1,21 @@
-FROM ubuntu:16.04
-
-ARG DEBIAN_FRONTEND=noninteractive
-ARG PIP_NO_CACHE_DIR=0
-
-RUN \
-    # Update the Ubuntu package index
-       apt-get update \
-    # Install the required Ubuntu packages
-    && apt-get install -y \
-        fontconfig \
-        python-pip \
-    # Remove the APT cache to reduce Docker image size
-    && rm -rf /var/lib/apt/lists/* \
-    # Install Pipenv
-    && pip install pipenv
+# FROM ubuntu:16.04
+FROM python:2-slim
 
 WORKDIR /app
+ENV DEBIAN_FRONTEND=noninteractive PIP_NO_CACHE_DIR=0
+ENTRYPOINT ["make"]
+
+RUN set -ex \
+	# Update the Ubuntu package index
+	&& apt-get update \
+	# Install the required Ubuntu packages
+	&& apt-get install -y fontconfig build-essential \
+	# Clean the APT cache to reduce Docker image size
+	&& apt-get clean \
```

Comment:
Can you please verify if `rm -rf /var/lib/apt/lists/*` and `apt-get clean` have the same effect? And if not, should we use both of them?

Reply:
Yes, I thought that apt-get clean will clean the package database cache as well, but apparently not. Both have been added.